docs/getting-started/next-steps.md

```diff
@@ -18,6 +18,7 @@ See the following guides for documentation specific to your use case:
 
 - [Serving Guide](../serving/README.md)
 - [Eventing Guide](../eventing/README.md)
+- [Next Steps With Functions](../function-next-steps/)
```

Comment:
I think the `function-next-steps` doc should maybe be a `../functions/README.md`, like Serving and Eventing?